cmd/bicep.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/massdriver-cloud/airlock/pkg/bicep"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdBicep() *cobra.Command {
+	bicepCmd := &cobra.Command{
+		Use:   "bicep",
+		Short: "Bicep translations",
+		Long:  helpdocs.MustRender("bicep"),
+	}
+
+	// Import
+	bicepInputCmd := &cobra.Command{
+		Use:   `input`,
+		Short: "Ingest a bicep template file and generate a JSON Schema",
+		Args:  cobra.ExactArgs(1),
+		Long:  helpdocs.MustRender("bicep/input"),
+		RunE: runBicepInput,
+	}
+
+	bicepCmd.AddCommand(bicepInputCmd)
+
+	return bicepCmd
+}
+
+func runBicepInput(cmd *cobra.Command, args []string) error {
+	schema, err := bicep.BicepToSchema(args[0])
+	if err != nil {
+		return err
+	}
+	fmt.Println(schema)
+	return nil
```

Comment:
How come this is returning `nil`?

Reply:
We need to return something for the `error`, and `nil` represents "no error occurred".